openlibrary/catalog/importer/load.py

```diff
@@ -47,7 +47,6 @@ def find_author(name, send_mail=True):
                     result += a['key'] + 'has bad type' + a + '\n'
             body += result
             addr = 'edward@archive.org'
```

Comment:
thanks for this PR! this email address should also be removed, and the line `subject = 'author query redirect: ' + repr(q['name'])` is not used either so can be deleted.

Reply:
I think it is also safe to rename `send_mail=True` to `log_errors=True` or similar and change the conditional at l.37 to avoid the now confusing variable name.

Reply:
Updated to reflect this change. As it is now, `send_mail` (and renamed `log_errors`) never is used that I could find. I think it could potentially be removed, but I'll leave that to your discretion / on a future PR.